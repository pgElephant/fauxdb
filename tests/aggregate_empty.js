db.runCommand({aggregate: "test", pipeline: []})
