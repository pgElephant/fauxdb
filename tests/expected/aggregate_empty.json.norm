{
  "cursor": {
    "firstBatch": [],
    "id": 0,
    "ns": "test.collection"
  },
  "ok": 1
}
