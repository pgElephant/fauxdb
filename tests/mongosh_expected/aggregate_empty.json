MongoServerError: The 'cursor' option is required, except for aggregate with the explain argument
