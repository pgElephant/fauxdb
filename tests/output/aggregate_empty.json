{
  ok: 1,
  cursor: { id: Long('0'), ns: 'test.collection', firstBatch: [] }
}
