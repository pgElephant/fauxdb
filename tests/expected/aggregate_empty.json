{
  "ok": 1,
  "cursor": {
    "id": 0,
    "ns": "test.collection",
    "firstBatch": []
  }
}
